GSA/digitalgov.gov
# Redirect blogs to intro translation technology

## Summary

Redirect https://digital.gov/2012/10/01/automated-translation-good-solution-or-not/ and https://digital.gov/2018/12/20/designing-for-translation/ to https://digital.gov/resources/introduction-to-translation-technology/

## Preview

[Link to Preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/cltm-redirect-intro_translation_technology/2012/10/01/automated-translation-good-solution-or-not/) and [Link to Preview] (https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/cltm-redirect-intro_translation_technology/2018/12/20/designing-for-translation/)



## Solution

Provide a summary of the solution this PR offers.




## How To Test

1. Go to https://digital.gov/2012/10/01/automated-translation-good-solution-or-not/ and https://digital.gov/2018/12/20/designing-for-translation/
2. Ensure they both redirect to https://digital.gov/resources/introduction-to-translation-technology/
